src/glim/mapping/sub_map.cpp

```diff
@@ -85,7 +85,7 @@ SubMap::Ptr SubMap::load(const std::string& path, const int start_from_submap_id
 
   std::string token;
   ifs >> token >> submap->id;
-  submap->id += start_from_submap_id;
+  submap->id;
```

Comment:
am i correct to assume that submap->id is either not correlated to the rekeyed frame_id, or not necessary to be correct?
Also, this line does nothing now.